sickbeard/subtitles.py

```diff
@@ -531,8 +531,8 @@ def get_video(video_path, subtitles_dir=None, subtitles=True, embedded_subtitles
         return video
 
     try:
-        video_path = encode(video_path)
-        subtitles_dir = encode(subtitles_dir or get_subtitles_dir(video_path))
+        video_path = unicode(video_path)
+        subtitles_dir = unicode(subtitles_dir or get_subtitles_dir(video_path))
 
```

Comment:
Then we need to do a type-check because the errors occur with latin characters when they are attempted to be encoded more than once.